shutdown_instance.go

```diff
@@ -72,7 +72,7 @@ func ReceiveEvent(ctx context.Context, msg *pubsub.Message) error {
 		result = multierror.Append(result, err)
 		log.Printf("Some error occured in stopping gce instances: %v", err)
 	}
-	report.Show(rpt)
+	report.Show(rpt, "InstanceGroup")
```

Comment:
🔧Struct ShutdownReport should have instance_type.
Once you're not enum, you can use String.

```go
type ShutdownReport struct {
       // Any values: InstanceGroup, Computeengine, SQL
       InstanceType string
	// shutdown resource names
	DoneResources []string
	// already stopped resource names
	AlreadyShutdownResources []string
}
```